.travis.yml

```diff
@@ -3,19 +3,19 @@ sudo: required
 language: crystal
 
 services:
-  - postgres
+  - postgresql
   - redis-server
+
 cache:
```

Comment:
We should cache `.cache/crystal` as well, WDYT?

Reply:
Why we should cache that directory?

Reply:
To reduce compilation times even more

I just realize we're checking `bin/#{APPNAME}` before compiling, so NVM, I'm ok without this :+1: 

Sorry for the blocking review :sweat_drops: